frontend/cypress/support/commands.ts

```diff
@@ -127,6 +127,18 @@ Cypress.Commands.add("kcLogout", () => {
   });
 });
 
+Cypress.Commands.add("hasErrorMessage", (inputs: Array<string>, toastText: string) => {
```

Comment:
This is slick, but might it obfuscate which input is raising the error versus the explicit checks? It's unclear to me if the input would get logged or if a screenshot would show that

Reply:
Great thought process, I had to try it out with an introduced missing validation message and see what it spat out - thankfully it does log the field it was testing (id=action-required-div in this example) before the error is thrown:

![image](https://github.com/user-attachments/assets/dad8a967-b979-4e58-b6cc-e89c80aafded)

Reply:
Awesome! Much cleaner then